examples/source_examples/toast/README.md

```diff
@@ -1,6 +1,133 @@
-# Toast Fivetran Connector
+# Toast Connector Example
 
-This is a custom [Fivetran Connector](https://fivetran.com/docs/connectors/connector-sdk) implementation to extract and sync data from the [Toast POS API](https://doc.toasttab.com/) into a destination warehouse. Toast is a restaurant management platform providing point-of-sale, labor, menu, and operational data.
+This example demonstrates how to use the Fivetran Connector SDK to integrate with the Toast platform API, enabling synchronization of restaurant data including orders, labor, configuration, and cash management information.
+
+## Requirements
+
+- Python 3.7 or later
+- Operating System:
+  - Windows 10 or later
+  - macOS 13 or later
+- Toast Developer Account
+- Valid Toast API credentials:
+  - Client ID
+  - Client Secret
+  - User Access Type
+  - Domain
+
+## Getting Started
+
+Follow the [Fivetran Connector SDK setup guide](https://fivetran.com/docs/connectors/connector-sdk/setup-guide) to get started.
+
+## Features
+
+- Comprehensive Toast API integration
+- Multi-endpoint data synchronization:
+  - Configuration data
+  - Labor information
+  - Cash management
+  - Order details
+- Automatic schema discovery
+- Efficient batch processing
+- State checkpointing for reliable resumption
+- Robust error handling and logging
+
+## Configuration File
+
+The connector requires the following parameters in the configuration file (`configuration.json`):
+
+```json
+{
+    "clientId": "<YOUR_CLIENT_ID>",
+    "clientSecret": "<YOUR_CLIENT_SECRET>",
+    "userAccessType": "<YOUR_ACCESS_TYPE>",
+    "domain": "<YOUR_DOMAIN>"
+}
+```
+
+## Authentication
+
+The connector uses OAuth2 client credentials flow:
+1. Obtain client credentials from Toast Developer Portal
+2. Configure the credentials in your configuration file
+3. The connector automatically handles token management and renewal
+4. Keep credentials secure and never commit them to version control
+
+## Data Handling
+
+### Endpoints and Data Types
+The connector syncs data from multiple Toast API endpoints:
+
+1. **Configuration Data**
```

Comment:
The endpoint list should a bullet list, not numbered one - no particular order/hierarchy are required/present here.

Reply:
@fivetran-elijahdavis I see we are not specifying tables here, but are talking about endpoints. Does this mean the Toast example syncs all data available for the endpoint that results in a dynamic schema?